DFC.Digital/DFC.Digital.Web.Sitefinity.Widgets/Mvc/Controllers/SetContentRelatedPageTitleController.cs

```diff
@@ -83,12 +83,18 @@ public ActionResult Index(string urlName)
             if (webAppContext.IsCategoryPage && !string.IsNullOrEmpty(urlName))
             {
                 var category = categoryRepo.GetByUrlName(urlName);
-                this.ViewBag.Title = $"{category.Title} {PageTitleSeperator} {PageTitleSuffix}";
+                if (category != null)
+                {
+                    this.ViewBag.Title = $"{category.Title} {PageTitleSeperator} {PageTitleSuffix}";
+                }
```

Comment:
What happens on the else part? What will citizen see when the category is null? I take it other widgets on the page would return a 404 response, however in isolation can we ensure the page title still makes sense?

Reply:
It would default to what has been set in the Page properties of the returned page , in this case it would be 404 as its used in conjunction with other widgets which return the 404 status code. In isolation (as the only widget on the page) it will just use the page title that is setup in the properties section. 

This widget is in itself an enhancer, without it default page title behaviour is kept at the page properties section